packages/app/src/specs/InlineRunAllSpecs.vue

```diff
@@ -4,7 +4,7 @@
     class="h-full truncate"
     data-cy="tooltip"
   >
-    <button>
+    <button @click.stop="emits('runAllSpecs')">
```

Comment:
This doesn't fix the a11y issue (probably still due to the parent's `click.capture`? Would need to look in more detail), but since there is a separate issue for that, we can continue there. It's still a better place for the click handler.

Reply:
https://github.com/cypress-io/cypress/issues/24762